README.md

```diff
@@ -74,8 +74,8 @@ If an Absence.io iCal URL is provided, users that are out-of-office when a pull
 
 ## 🚧 Dependencies
 
-* Elixir (`1.7.4`)
-* Erlang (`21.1.2`)
+* Elixir (`1.8.1`)
+* Erlang (`21.2.6`)
```

Comment:
Weren't we at `21.2.7` with our boilerplate?

Reply:
Right! I wasn't able to install it locally yesterday with `asdf`, I forgot to set it back to `21.2.7`…